pages/index.js

```diff
@@ -72,7 +72,7 @@ export default function Index() {
           <div className="flex justify-center text-xl my-10">
             <a
               className="nes-btn is-primary font-small mx-5"
-              href="https://github.com/opentech-pe/opentech.pe/issues/new?assignees=jhonfitzgerald%2C+paulotijero%2C+JimmyLoloy98&labels=cfp&template=propuesta-de-charla.md&title=T%C3%ADtulo+de+tu+charla+.+.+."
+              href="https://github.com/opentech-pe/opentech.pe/issues/new?template=propuesta-de-charla.md&title=T%C3%ADtulo+de+tu+charla"
```

Comment:
Igualmente por acá `&labels=cfp` esto es para agregar el label de `Call for Papers`